python/gui/auto_generated/qgsnewhttpconnection.sip.in

```diff
@@ -43,7 +43,7 @@ information for an HTTP Server for WMS, etc.
 
     QgsNewHttpConnection( QWidget *parent /TransferThis/ = 0,
                           QgsNewHttpConnection::ConnectionTypes types = ConnectionWms,
-                          const QString &baseKey = "qgis/connections-wms/",
+                          const QString &serviceName = "WMS",
```

Comment:
This is an api break, isn't it?

Reply:
no, this is handled here https://github.com/qgis/QGIS/pull/48192/files?w=1#diff-72f4e9193431ef42d291a483c045825bfacb9df7e5691fd065e091ec47d636b8R40

Reply:
![image](https://user-images.githubusercontent.com/127259/165238685-c1c29b88-dbce-48b8-be24-e41a4c1b2cdf.png)

Reply:
I mean the rename of the keyword -- if some code is using `QgsNewHttpConnection(..., baseKey='xxxxx')` then it'll now raise an exception.

Reply:
you're right. I've renamed the arg for python